scripts/check_public_api_docstrings.py

```diff
@@ -25,21 +33,22 @@ class DocstringError:
         name: name of the class, method, function - empty if module
         filepath_relative_to_repo_root: location of error
         error: stripped string of error
-        raw_error_first_line: raw string of error first line
-        raw_error_second_line: raw string of error second line
+        raw_error: raw string of error
         line_number: line number of error within file
 
     """
 
     name: str
     filepath_relative_to_repo_root: pathlib.Path
     error: str
-    raw_error_first_line: str
-    raw_error_second_line: str
+    raw_error: str
     line_number: int
 
+    def __str__(self):
+        return self.raw_error
```

Comment:
Printing the raw error instead of the class definition so you get a similar output to running the linters on their own.